x-pack/legacy/plugins/fleet/server/mappings.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const mappings = {
+  agents: {
+    properties: {
+      shared_id: {
+        type: 'text',
```

Comment:
Here we probably want to use keyword
```suggestion
        type: 'keyword',
```